jd-geek-source/src/app/app.module.ts

```diff
@@ -3,18 +3,49 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { BatmanComponent } from './batman/batman.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { BatmanComponent } from './components/batman/batman.component';
+import { TestDiplayComponent } from './components/test-diplay/test-diplay.component';
+import { ApiService } from './services/api.services';
+import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
+import { JwtInterceptor } from './interceptors/jwt.interceptor';
+import { HeaderComponent } from './components/header/header.component';
+import { FooterComponent } from './components/footer/footer.component';
+import { ButtonComponent } from './components/button/button.component';
+import { ProductShowcaseComponent } from './components/product-showcase/product-showcase.component';
+import { SearchComponent } from './components/search/search.component';
+import { ShopByCategoryComponent } from './components/shop-by-category/shop-by-category.component';
+import { HeroImageComponent } from './components/hero-image/hero-image.component';
+import { FeaturedProductsComponent } from './components/featured-products/featured-products.component';
+import { DailyDealsComponent } from './components/daily-deals/daily-deals.component';
+import { HottestDealsComponent } from './components/hottest-deals/hottest-deals.component';
```

Comment:
always try to group your imports together.  sometimes it's alphabetical, but in angular, typically you should list them in the following order:

- imports from angular itself.
-----------------------
- third party imports such as angular material, lodash, rxjs, etc.
----------------------
- custom imports such as your own models, modules, services, components, etc